homeassistant/components/poolsense/binary_sensor.py

```diff
@@ -0,0 +1,75 @@
+"""Support for PoolSense binary sensors."""
+import logging
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_PROBLEM,
+    BinarySensorEntity,
+)
+from homeassistant.const import CONF_EMAIL
+
+from . import PoolSenseEntity
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+BINARY_SENSORS = {
+    "pH Status": {
+        "unit": None,
+        "icon": "mdi:pool",
+        "name": "pH Status",
+        "device_class": DEVICE_CLASS_PROBLEM,
+    },
+    "Chlorine Status": {
+        "unit": None,
+        "icon": "mdi:pool",
+        "name": "Chlorine Status",
+        "device_class": DEVICE_CLASS_PROBLEM,
+    },
+}
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Defer sensor setup to the shared sensor module."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    binary_sensors_list = []
+    for binary_sensor in BINARY_SENSORS:
+        binary_sensors_list.append(
+            PoolSenseBinarySensor(
+                coordinator, config_entry.data[CONF_EMAIL], binary_sensor
+            )
+        )
+
+    async_add_entities(binary_sensors_list, False)
+
+
+class PoolSenseBinarySensor(PoolSenseEntity, BinarySensorEntity):
+    """Representation of PoolSense binary sensors."""
+
+    @property
+    def is_on(self):
+        """Return true if the binary sensor is on."""
+        return self.coordinator.data[self.info_type] == "red"
+
+    @property
+    def icon(self):
```

Comment:
device class problem has its own icons (handled by frontend) for consistency.

hass:shield-check and hass:alert

So going to remove these icons overrrides